test/main_unit.C

```diff
@@ -151,5 +151,12 @@ int main(void)
 
   fsdt / fsdt;
 
+  long int divl = 1;
+  long long int divll = 1;
+  FullFirstDerivType dbl = 2.0;
+  FullFirstDerivType r1 = dbl/divl;
+  FullFirstDerivType r2 = dbl/divll;
```

Comment:
Maybe you can point me to a better place to test the new division by `long int` and `long long int`...

Reply:
Nah, this is ideal.